### ISSUE TITLE
Radeon R9 390

### ISSUE THREAD

**Issue by NoxiousGit** [Vipeax/MSI-LED-Tool#16]
![msi led tool](https://cloud.githubusercontent.com/assets/25221797/22102901/402af8f2-de8d-11e6-8043-7e764a18c1f6.png)
Thank you for you work!
(No color on that card, only white led that I want breathing posible with msi gaming app, but its the gaming app..)

**Comment by NoxiousGit** [comment]
Btw just saying, I did try to set the Overwrite security check to true, now the app launch at the back, but nothing happen with the LED, here is my config file:
{  
   "R":255,
   "G":255,
   "B":255,   
   "AnimationType":2,
   "TemperatureLowerLimit":45,
   "TemperatureUpperLimit":87,
   "OverwriteSecurityChecks":true
}

**Comment by cyntrifical** [comment]
I don't think it supports this card. I have two r9 390's running crossfire and using MSI's Gaming APP tool caused one of my cards LED's to turn off completely while my secondary card stayed on......then tried to install this and it said the same thing.
